mesonbuild/dependencies/base.py

```diff
@@ -1131,9 +1131,15 @@ def search():
         cm_path = stringlistify(extract_as_list(kwargs, 'cmake_module_path'))
         cm_args = stringlistify(extract_as_list(kwargs, 'cmake_args'))
         cm_path = [x if os.path.isabs(x) else os.path.join(environment.get_source_dir(), x) for x in cm_path]
-        if cm_path:
```

Comment:
`CMAKE_MODULE_PATH` is still required for custom Find<Dep>.cmake files (see CI failures).

Reply:
oops, that was not supposed to be in there. that's leftover from the original attempt I made to use CMAKE_MODULE_PATH. Fixed now.